src/frontend/components/PublicVideoLiveJitsi/index.spec.tsx

```diff
@@ -31,22 +47,87 @@ jest.mock('../../data/appData', () => ({
   },
 }));
 
-const mockExecuteCommand = jest.fn();
-const mockJitsi = jest.fn().mockImplementation(() => ({
-  executeCommand: mockExecuteCommand,
-  addListener: jest.fn(),
+jest.mock('components/StudentLiveWrapper', () => ({
+  LiveVideoWrapper: jest.fn(),
 }));
 
-describe('<PublicVideoLiveJitsi />', () => {
-  it('renders jitsi and the leave button', () => {
-    global.JitsiMeetExternalAPI = mockJitsi;
+jest.mock('utils/conversejs/converse', () => ({
+  initConverse: jest.fn(() => jest.fn()),
+}));
+
+describe('<PublicVideoLiveJitsi /> unit tests', () => {
+  beforeAll(() => {
+    (LiveVideoWrapper as any).mockImplementation(() => (
```

Comment:
I suggest you to use before the `jest.MockedFunction` helper to avoid using `as any`.

```
const mockLiveVideoWrapper = LiveVideoWrapper as jest.MockedFunction<typeof LiveVideoWrapper>
```

And then use `mockLiveVideoWrapper` everywhere you need it.